python-scripts/update_hearings_with_caseReferenceNumber.py

```
@@ -0,0 +1,182 @@
+import requests
+from datetime import datetime
+import csv
+
+
+env_url = "https://dristi-kerala-uat.pucar.org"
```

Pylint (3.3.7)

[convention] Constant name "env_url" doesn't conform to UPPER_CASE naming style

(C0103)